modules/nwtc-library/tests/test_NWTC_IO_FileInfo.F90

```diff
@@ -62,7 +62,8 @@ subroutine test_initfileinfo2()
 
     integer(IntKi) :: error_status
     character(ErrMsgLen) :: error_message
-    character(1025) :: input_strings(num_lines)
+    character(MaxFileInfoLineLen*2) :: input_strings(num_lines)
```

Comment:
Why the `*2` here?

Reply:
Just to make sure it was longer than the standard size.  We could make it `MaxFileInfoLineLen+1` instead.

Reply:
No problem, this makes sense.